### ISSUE TITLE
Set the namespace for the ThunderSVM DataSet class

### ISSUE THREAD

**Issue by IaBogdi** [Xtra-Computing/thundersvm#166]
Dear developers,

first of all, thank you for the GPU implementation of SVM, it works really good (especially for the prediction task). 
I have the following problem: I use ThunderSVM as .dll in my C++ project and got a namespace conflict between libraries: the class DataSet has the same name in ThunderSVM and other libraries which I used and they're not written by me.  Could you create a namespace for ThunderSVM classes  so it'll be safe to use?

**Comment by zeyiwen** [comment]
Thanks for the feedback! We plan to build a common utility for our projects including ThunderSVM, and we will put your suggestion into our todo list. Please stay tuned.

**Comment by emmenlau** [comment]
Dear @zeyiwen , we also consider this a nice and relevant addition, to have namespaces for the C++ interface.

Would you accept contributions or pull requests for this? What namespace are you planning to use, something like `tsvm` or `thundersvm`? Would it be acceptable to add this to a few classes first and extend it over time? `Dataset` seems like a good start, as the name may be prone to name collisions :-)

**Comment by zeyiwen** [comment]
Yes, we welcome contributions. I think ``thundersvm`` would be better for the namespace. Please feel free to start from the ``Dataset`` class.
